README.rst

```diff
@@ -102,3 +102,41 @@ Perform a query
 
     for row in rows:
         print(row.name)
+
+Instrumenting With OpenTelemetry
+--------------------------------
+
+This application uses `OpenTelemetry`_ to output tracing data from
+API calls to BigQuery. To enable OpenTelemetry tracing in
+the BigQuery client the following PyPI packages need to be installed:
+
+.. _OpenTelemetry: https://opentelemetry.io
+
+.. code-block:: console
+
+    pip install opentelemetry-api opentelemetry-sdk opentelemetry-instrumentation opentelemetry-exporter-google-cloud
```

Comment:
```suggestion
    pip install google-cloud-bigquery[opentelemetry]
```

Reply:
I'd still prefer if these dependencies were defined in `extras`. You can add a line to exclude from `all` since it's not supported on Python 2.7.